products/website/src/pages/index.js

```diff
@@ -52,9 +52,11 @@ function Home() {
               A pluggable open-source platform for visualizing and integrating
               biological data
             </p>
-            <Button variant="contained" color="secondary" disableElevation>
-              Download latest release
-            </Button>
+            <Link href="/jb2/blog/" target="_blank">
+              <Button variant="contained" color="secondary" disableElevation>
+                Download latest release
+              </Button>
+            </Link>
```

Comment:
I think `Link` is for Typography-based links, could add `href` and `target` to `Button` itself here.